### PR TITLE
Added a basic cockpit example and a cmake routine to build static files

### PR DESCRIPTION
**Pull request by SindriTh**

### CODE REVIEW

File: cockpit/IPC/src/App.tsx [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+      console.log(data)
+    })
+  }
+
+
+  const sendDbusMessage = () => {
+    console.log("Sending dbus message")
+
+    // @ts-ignore
+    const dbus = cockpit.dbus("org.freedesktop.Notifications", { bus: "session" });
```
Comment (jbbjarnason) [inline cockpit/IPC/src/App.tsx:67]: Future thought, can we make a interface class which specifies the dbus interface to tfc? One common place.

**Review by jbbjarnason — COMMENTED** [review]
nice work

File: cockpit/IPC/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# npm i
+# react-scripts build
+
+
+add_custom_command(
+   OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/build
```
[the remark this answers is not in the document]
Comment (omarhogni) [reply cockpit/IPC/CMakeLists.txt:6]: BINARY_DIR is already cockpit/IPC.

File: cockpit/IPC/package.json [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    "devbuild": "GENERATE_SOURCEMAP=true BUILD_PATH='./build' react-scripts build && cp manifest.json ./build/manifest.json",
+    "eject": "react-scripts eject"
+  },
+  "eslintConfig": {
+    "extends": [
+      "react-app",
+      "react-app/jest"
+    ]
+  },
+  "browserslist": {
```
[the remark this answers is not in the document]
Comment (omarhogni) [reply cockpit/IPC/package.json:40]: Removed.

**Review by omarhogni — APPROVED** [review]

**Pull request merged**